### PR TITLE
Remove Unneeded Escaped param for DLC Cli

### PR DESCRIPTION
**Pull request by benthecarman**
Same as #2178 but for the DLC branch and its exclusive cli commands

### CODE REVIEW

File: app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCMessage.scala [hunk 1]
```diff
@@ -178,9 +178,9 @@ object DLCMessage {
         fundingInputs.map { input =>
           val obj = mutable.LinkedHashMap(
             "prevTx" -> Str(input.prevTx.hex),
-            "prevTxVout" -> Num(input.prevTxVout.toInt),
-            "sequence" -> Num(input.sequence.toInt),
-            "maxWitnessLength" -> Num(input.maxWitnessLen.toInt)
+            "prevTxVout" -> Num(input.prevTxVout.toLong.toDouble),
```
Comment (Christewart) [inline app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCMessage.scala:181]: Wait so why is this needed?

**Review by Christewart — APPROVED** [review]

File: app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCMessage.scala (hunk 1 above)
Comment (benthecarman) [reply app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCMessage.scala:181]: `toInt` wasn't big enough to cover for all `UInt32` and and `Num()` can't take a long, so had to a double
